### ISSUE TITLE
Support Composite Primary Keys as Tuples

### ISSUE THREAD

**Issue by bezborodow** [bezborodow/psyqlepg#1]
Effected methods:

```
src/psyqlepg/__init__.py:57:def selectone(conn, table, primary_key, identifier):
src/psyqlepg/__init__.py:110:def update(conn, table, primary_key, identifier, **kwargs):
src/psyqlepg/__init__.py:157:    def get(cls, conn, identifier, key=None):
src/psyqlepg/__init__.py:169:    def update(cls, conn, identifier, key=None, **kwargs):
```

**Comment by bezborodow** [comment]
Released 0.0.8.

**Comment by bezborodow** [comment]
Further changes in release 0.1.0.
